public/kibi_timeline.js

```diff
@@ -149,25 +149,29 @@ define(function (require) {
         const groupId = group.id;
         const groupColor = group.color;
 
+        searchSource.highlight({
```

Comment:
Good point about the angular filter. However, I would still use the highlight tags from kibana, since it seems that they had some trouble with the `<em>` tags https://github.com/sirensolutions/kibi/blob/master/src/ui/public/highlight/highlight_tags.js#L4